hail/python/hail/expr/expressions/typed_expressions.py

```diff
@@ -3058,6 +3058,43 @@ def sequence_context(self, before=0, after=0):
             raise TypeError("Reference genome '{}' does not have a sequence loaded. Use 'add_sequence' to load the sequence from a FASTA file.".format(rg.name))
         return hl.get_sequence(self.contig, self.position, before, after, rg)
 
+    @typecheck_method(before=expr_int32, after=expr_int32)
+    def window(self, before, after):
+        """Returns an interval of a specified number of bases around the locus.
+
+        Examples
+        --------
+        Create a window of one megabase around a locus:
```

Comment:
quibble, but isn't this a two mega base window centered on the locus? Do people usually refer to the radius?

Reply:
I have no idea